src/Microsoft.DocAsCode.AzureMarkdownRewriters/AzureBlockRules/AzureMigrationHtmlMetadataBlockRule.cs

```diff
@@ -82,7 +82,13 @@ private AzureHtmlMetadata GetAttributesFromHtmlContent(string htmlContent)
                 {
                     if (string.Equals(attribute.Name, "pageTitle", StringComparison.OrdinalIgnoreCase))
                     {
-                        attributes["title"] = attribute.Value;
+                        // Per Azure's request, migration script should convert "| Microsoft Azure" at the end of title to "| Microsoft Docs".
+                        var title = attribute.Value;
+                        if (title != null && title.EndsWith("| Microsoft Azure"))
+                        {
+                            title = title.Replace("| Microsoft Azure", "| Microsoft Docs");
```

Comment:
Use regular expression now.

---
In reply to: [85080171](https://github.com/dotnet/docfx/pull/826#discussion_r85080171) [](ancestors = 85080171)